Envelopes/cool_MK ShaperStutter.lua

```diff
@@ -0,0 +1,7140 @@
+
+leds_table[3] = Colored_Rect_top:new(981,5,2,20,  0.0,0.7,0.0,1 ) -- Light_Loop_on
+leds_table[4] = Colored_Rect_top:new(981,5,2,20,  0.5,0.5,0.5,0.5 ) -- Light_Loop_off
+
+leds_table[5] = Colored_Rect_top:new(921,5,2,20,  0.0,0.7,0.0,1 ) -- Light_Sync_on
+leds_table[6] = Colored_Rect_top:new(921,5,2,20,  0.5,0.5,0.5,0.5 ) -- Light_Sync_off
+
+leds_table[21] = Colored_Rect:new(571,410+corrY,2,18,  0.0,0.7,0.0,1 ) -- InverseEnv_On (Green indicator)
+leds_table[22] = Colored_Rect:new(571,410+corrY,2,18,  0.5,0.5,0.5,0.5 ) -- InverseEnv_Off (Grey indicator)
+
+local others_table = {Triangle, RandText, Q_Rnd_Linked, Q_Rnd_Linked2, Line, Line2, Loop_Dis}
+
+others_table[1] = Txt2:new(642,415+corrY2,55,18, 0.4,0.4,0.4,1, ">","Arial",20) --Triangle
+others_table[2] = Txt2:new(749,374+corrY2,55,18, 0.4,0.4,0.4,1, "Intensity","Arial",10) --RandText
+
+others_table[3] = Line_colored:new(482,375+corrY,152,18,  0.7,0.5,0.1,1) --| Q_Rnd_Linked (Bracket)
+others_table[4] = Line2:new(480,380+corrY,156,18,  0.177,0.177,0.177,1)--| Q_Rnd_Linked2 (Bracket fill)
+
+others_table[5] = Line:new(677,404+corrY,82,6) --Line (Preset/Velocity Bracket)
+others_table[6] = Line2:new(677,407+corrY,82,4,  0.177,0.177,0.177,1)--Line2 (Preset/Velocity Bracket fill)
+others_table[7] = Colored_Rect_top:new(10,28,1024,15,  0.23,0.23,0.23,0.5)--Loop_Dis (Loop Disable fill)
+others_table[8] = Line:new(771,404+corrY,61,6) --Line (Mode Bracket)
+others_table[9] = Line2:new(771,407+corrY,61,4,  0.177,0.177,0.177,1)--Line2 (Mode Bracket fill)
+
+local Frame_Sync_TB = {leds_table[5]}
+local Frame_Sync_TB2 = {leds_table[6]}
+local Frame_Loop_TB = {leds_table[3]}
+local Frame_Loop_TB2 = {leds_table[4], others_table[7]}
+local Frame_TB = {elm_table[1], elm_table[2], elm_table[3]} 
+local FrameR_TB = {others_table[5], others_table[6], others_table[8], others_table[9]}
+local FrameQR_Link_TB = {others_table[3],others_table[4]}
+local Frame_TB1 = {leds_table[2]}
+local Frame_TB2 = {elm_table[5], leds_table[1]} -- Grid mode
+local Frame_TB2_Trigg = {elm_table[4]}
+
+local Grid1_Led_TB = {elm_table[8]}
+local Grid2_Led_TB = {elm_table[9]}
+local Grid4_Led_TB = {elm_table[10]}
+local Grid8_Led_TB = {elm_table[11]}
+local Grid16_Led_TB = {elm_table[12]}
+local Grid32_Led_TB = {elm_table[13]}
+local Grid64_Led_TB = {elm_table[14]}
+local GridT_Led_TB = {elm_table[15]}
+local Swing_Led_TB = {elm_table[16]}
+
+local Triangle_TB = {others_table[1]}
+local RandText_TB = {others_table[2]}
+
+local InvertEnvOn_TB = {} --leds_table[21]
+local InvertEnvOff_TB = {} --leds_table[22]
+
+local Transient_Fill_TB = {elm_table[17]}
+local Grid_Fill_TB = {elm_table[18]}
+
+---------------------------------------------------------------
+---  Create Menu Settings   -----------------------------
+---------------------------------------------------------------
+
+function OpenURL(url)
+  local OS = r.GetOS()
+  if OS == "OSX32" or OS == "OSX64" then
+    os.execute('open "" "' .. url .. '"')
```

Comment:
Note that this opens an extra Finder window (due to an extra first argument) in addition to opening the URL. (And this `OpenURL` doesn't work on Linux).

I'd recommend using `CF_ShellExecute` from SWS instead, which is cross-platform.